middleware/ratelimiter/ratelimiter.go

```diff
@@ -19,6 +19,8 @@ type LimiterConfig struct {
 	// Unique identifier for requestors. Can be userID or IP
 	// Defaults to Remote Address if it is empty
 	ClientID interface{}
+
+	secondsToQuotaReset float64
```

Comment:
This is not what I meant. What you've done is a workaround only used for testing. It defeats the whole purpose of testing.

Here is an example of what I was thinking about:
```go
request := suite.CreateTestRequest(nil)
clientID := request.RemoteAddress()
resetTime := time.Now().Add(5 * time.Second)

store := newLimiterStore()
limiter := store.get(clientID)
limiter.resetsAt = resetTime

middleware := newWithStore(store)
result := suite.Middleware(...)
suite.Equal(resetTime, result.Header.Get("RateLimit-Reset")
```

This would need some very light refactoring so we're able to pass a custom limiter store:
```go
func New(configFn LimiterConfigFunc) goyave.Middleware {
	return newWithStore(configFn, newLimiterStore())
}

func newWithStore(configFn LimiterConfigFunc, lstore limiterStore) goyave.Middleware {
	return func(next goyave.Handler) goyave.Handler {
		return func(response *goyave.Response, request *goyave.Request) {
			//...
		}
	}
}
```